assets/scss/texts.scss

```diff
@@ -21,11 +21,16 @@ header .col {
 .text-page {
   background: $white;
   margin-top: -3rem;
-  
+  overflow-x: hidden;
```

Comment:
Remove this here, it is not necessary, see suggestion in BlogPost.vue

Reply:
ok